src/utils/__tests__/index.ts

```diff
@@ -2099,6 +2102,28 @@ describe('txTagToProtocolOp', () => {
   });
 });
 
+describe('txTagToExtrinsicIdentifier and extrinsicIdentifierToTxTag', () => {
```

Comment:
Use examples where both the module and the call names have more than one word in them